src/main/java/seedu/address/model/person/Note.java

```diff
@@ -9,6 +9,7 @@ public class Note {
 
     public static final String MESSAGE_CONSTRAINTS = "Notes should be alphanumeric.";
     public static final String DUPLICATE_MESSAGE_CONSTRAINTS = "There is already an existing note with this name.";
+    public static final String NO_EXISTING_NOTE = "There is no note with this index.";
```

Comment:
I think this can be moved to Messages, similar to `Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX`.